.github/CODEOWNERS

```diff
@@ -260,6 +260,9 @@
 # PRLabel: %Web Apps
 /specification/web/ @naveedaz @Azure/azure-app-service-control-plane
 
+# PRLabel: %Video Indexer
+/specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
```

Comment:
```
Error(s) on line 264
Source Line: /specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
  -dazilb is not a public member of Azure.
  -tshoham is not a public member of Azure.
  -omerhaimov is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
  -bermanraz is not a public member of Azure.
  -Einav94 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.

There were linter errors. Please visit https://aka.ms/azsdk/codeownersLinterErrors for guidance on how to handle them.
```

Reply:
dazilb is a public member 
<img width="1548" alt="image" src="https://github.com/user-attachments/assets/c9420f07-eb7c-4753-bc97-dc804444b6ba">

is this the correct group?

Reply:
```
Error(s) on line 263
Source Line: # PRLabel: %Video Indexer
  -'Video Indexer' is not a valid label for this repository.

Error(s) on line 264
Source Line: /specification/vi/ @dazilb @tshoham @omerhaimov @bermanraz @Einav94
  -tshoham is not a public member of Azure.
  -omerhaimov is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.
  -Einav94 is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.

There were linter errors. Please visit https://aka.ms/azsdk/codeownersLinterErrors for guidance on how to handle them.
```

@dazlib looks like your account is corrected but there are still other issues.

Reply:
my account (and everyone else''s) is public, so it might be a problem with the linter

Reply:
@weshaggard thanks! q: why is Video Indexer an invalid label?

Reply:
That label doesn't exist in the repo https://github.com/Azure/azure-rest-api-specs/labels?q=Video+Indexer

Reply:
I see, thanks so much :)

Reply:
I added it but it's still failing. How long does it take to propagate?

Reply:
@weshaggard all checks have passed, can you please approve? :)